src/DynamoCore/Migration/Migration.cs

```diff
@@ -41,6 +41,9 @@ public Migration(Version v, Action upgrade)
         }
     }
 
+    /// <summary>
+    /// A class which provides functionality for managing migration of Dynamo workspaces
```

Comment:
This class contains methods and properties used for migration of Dynamo workspaces.